2048/2048/build.gradle

```diff
@@ -1,11 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
+    ndkVersion "17.2.4988734"
 
     defaultConfig {
         minSdkVersion 23
-        targetSdkVersion 28
+        targetSdkVersion 29
```

Comment:
Why does API Level 29 need to be targeted?

Reply:
29 doesn't need to be targeted. Nonetheless I think it would be good to know if it breaks anything.

Reply:
Ah, unfortunately it will mess up the activity management. This came up in #integrations on Monday: https://skillz.slack.com/archives/G26HBS6NN/p1583539111132400